cep18/src/main.rs

```diff
@@ -359,12 +360,37 @@ pub extern "C" fn change_security() {
         sec_change_map: badge_map,
     }));
 }
+pub fn upgrade(name: &str) {
+    let entry_points = generate_entry_points();
 
-#[no_mangle]
-pub extern "C" fn migrate() {}
+    let contract_package_hash = runtime::get_key(&format!("{HASH_KEY_NAME_PREFIX}{name}"))
+        .unwrap_or_revert()
+        .into_hash()
+        .map(ContractPackageHash::new)
+        .unwrap_or_revert_with(Cep18Error::MissingPackageHashForUpgrade);
 
-pub fn install_contract() {
-    let name: String = runtime::get_named_arg(NAME);
+    let previous_contract_version = runtime::get_key(&format!("{CONTRACT_NAME_PREFIX}{name}"))
```

Comment:
Hi, would you mind changing the name of that var to previous_contract_hash ? because it is not a numerical version as the other contract_version